### ISSUE TITLE
Chrome's autofill recommendations close HoverCard.Content

### ISSUE THREAD

**Issue by Strahinja-Samardzija** [radix-ui/primitives#2502]
### Current Behavior

When autofill dropdown for an <input> pops up in Google Chrome, selecting an option or hovering over it closes the HoverCard.Content.

Also it is almost impossible to use devtools for HoverCard.Content because it closes when mouse hovers away to devtools tab.

### Expected behavior

It treats Chrome's menus and other option pop-ups as children of HoverCard.Content and not lose focus-within and set data-state="close".

### Reproducible example

Put a form with input type="username" or "email" inside HoverCard.Content. 
Try to select or hover over an autofill recommendation.

**Comment by kotAPI** [comment]
Yeah, this is an annoying behavior of the Select, it's not a native implementation, so it doesn't work like it.
